application/models/testutils_test.go

```diff
@@ -213,10 +213,17 @@ func createPotentialProviderFixtures(tx *pop.Connection, nRequests, nProviders i
 // createLocationFixtures generates any number of location records for testing.
 func createLocationFixtures(tx *pop.Connection, n int) Locations {
 	countries := []string{"US", "CA", "MX", "TH", "FR", "PG"}
+	states := []string{"FL", "ON", "", "", "", ""}
+	cities := []string{"Miami", "Toronto", "Mexico City", "Chiang Mai", "Paris", "Port Moresby"}
 	locations := make(Locations, n)
+
+	/* #nosec */
 	for i := range locations {
+		randInt := rand.Intn(6)
 		locations[i] = Location{
-			Country:     countries[rand.Intn(6)],
+			Country:     countries[randInt],
+			State:       states[randInt],
```

Comment:
Looks like we'll get locations like "Mexico City, FL, FR" Reminds me of those kids' books with mix-and-match animal parts, haha!

Reply:
I intended to make the city, state and country all correspond with each other.  This does that doesn't it?  (I.e. Miami will only go with Florida and the US)

Reply:
Oh, yeah, sorry. I just saw `randInt` on them and my brain though they were all separately random.